package proto_ff;

import "yd_fieldoptions.proto";


message vipvip
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 level = 2[(yd_fieldoptions.field_cname) = "vip等级"];
	optional int32 exp = 3[(yd_fieldoptions.field_cname) = "经验值"];
	optional int32 privilege_group = 4[(yd_fieldoptions.field_cname) = "特权组"];
}


message Sheet_vipvip
{
	repeated vipvip vipvip_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message vipprivilege
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 group = 2[(yd_fieldoptions.field_cname) = "group"];
	optional int32 type = 3[(yd_fieldoptions.field_cname) = "特权类型"];
	optional int32 arg = 4[(yd_fieldoptions.field_cname) = "功能参数"];
	optional string args = 5[(yd_fieldoptions.field_cname) = "整数功能参数", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_vipprivilege
{
	repeated vipprivilege vipprivilege_List = 1[(yd_fieldoptions.field_arysize)=60];
}
